File: HLCObjKotlinAct4/src/Java/Act12.java
package Java;

import java.util.Scanner;

public class Act12 {
    public static void main(String[] args) {
        int n,reverse = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("Escribe un numero entero");
        n = sc.nextInt();
        sc.nextLine();
        while (n != 0) {
            reverse = reverse * 10 + n % 10;
            n /= 10;
        }
        System.out.println(reverse);

    }
}

File: HLCObjKotlinAct4/src/Kotlin/Act12.kt
package Kotlin

fun main() {
    var reverse = 0
    println("Escribe un numero entero")
    var n: Int = readln().toInt()
    while (n != 0) {
        reverse = reverse * 10 + n % 10
        n /= 10
    }
    println(reverse)
}

